packages/developer-console-spa/src/components/ConfigureFeedback.tsx

```diff
@@ -1,16 +1,155 @@
-import { Card, CardBody, Title } from '@patternfly/react-core';
-import UnderDevelopment from './UnderDevelopment';
+import { Card, CardBody, Stack, StackItem, Form, FormGroup, Radio, Grid, GridItem, TextInput, Title, ActionGroup, Button, Popover, Tooltip } from '@patternfly/react-core';
+import { useContext, useEffect, useState } from 'react';
+import { AppContext } from '../context/AppContext';
+import useFeedbackConfig from '../hooks/useFeedbackConfig';
+import Header from './Header';
 
 function ConfigureFeedback ( props: any ) {
+  const { appId, app } = useContext( AppContext );
+  const { feedbackConfig } = useFeedbackConfig( appId );
+  const [ editableFeedbackConfig, setEditableFeedbackConfig ] = useState( feedbackConfig );
+  const [ canSubmit, setCanSubmit ] = useState( false );
+
+  useEffect( () => {
+    setEditableFeedbackConfig( feedbackConfig );
+  }, [ feedbackConfig ] );
+
+  useEffect( () => {
+    setCanSubmit(
+      !!editableFeedbackConfig.sourceType
+      && !!editableFeedbackConfig.sourceApiUrl
+      && editableFeedbackConfig.feedbackEmail
+      && isSourceEnabled( editableFeedbackConfig.sourceType )
+    );
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [ editableFeedbackConfig ] );
+
+  const feedbackTargets = [
+    { id: 'JIRA', name: 'JIRA', description: 'projects.engineering.redhat.com', url: 'projects.engineering.redhat.com', isEnabled: true },
+    { id: 'GITLAB', name: 'Gitlab', description: 'gitlab.cee.redhat.com', url: 'gitlab.cee.redhat.com', isEnabled: false },
+    { id: 'GITHUB', name: 'GitHub', description: 'github.com', url: 'github.com', isEnabled: false },
+    { id: 'EMAIL', name: 'Other', description: 'Other JIRA, Gitlab instances or Email', url: null, isEnabled: false },
+  ];
+
+  const handleSourceTypeSelect = (target: any) => {
+    setEditableFeedbackConfig( { ...editableFeedbackConfig, sourceType: target.id, sourceApiUrl: target.url } );
+  }
+  const isSourceTypeSelected = ( sourceType: string ) => {
+    return editableFeedbackConfig.sourceType === sourceType;
+  };
+  const isSourceEnabled = ( sourceType: string ) => {
+    return feedbackTargets.findIndex( target => target.id === sourceType && target.isEnabled ) !== -1;
+  }
+
+  const saveFeedbackConfig = (event: any) => {
+    event.preventDefault();
+
+    if ( !canSubmit ) {
+      return;
+    }
+
+    console.log( feedbackConfig );
+    console.log( { ...app, feedback: editableFeedbackConfig } );
```

Comment:
```suggestion
```

Reply:
I was still working on that part, hence the draft PR.
I've updated the commits.